package.json

```diff
@@ -21,8 +21,8 @@
   },
   "lint-staged": {
     "*.{js,jsx}": [
-      "eslint --fix",
       "prettier --write",
+      "eslint --fix",
```

Comment:
inverted these two, right now you still get a lot of non fixed code because of the order